server/utils/__tests__/update-uids.test.js

```diff
@@ -31,6 +18,7 @@ const defaultUid = 'unique-uid'
 const contentManagerPluginMock = {
   services: {
     uid: () => ({
+      // eslint-disable-next-line no-unused-vars
       generateUIDField({ contentTypeUID, field, data }) {
```

Comment:
I believe we can remove the unused vars here right?
```suggestion
      generateUIDField() {
```